src/components/round-buttons/RoundButton.jsx

```diff
@@ -0,0 +1,92 @@
+// @flow strict
+
+import React from 'react';
+import cx from 'classnames';
+import Icon, {ICON_COLOR, TYPE} from '../icons/Icon';
+import {ROUND_BUTTON_SIZE, ROUND_BUTTON_COLOR} from './roundButtonsConsts';
+
+export {ROUND_BUTTON_SIZE, TYPE as ROUND_BUTTON_ICON_TYPE, ROUND_BUTTON_COLOR};
+
+export type RoundButtonPropsType = {
+  /**
+   * Sizes for round buttons
+   * @example <RoundButton size="small" />
+   * @see size="small" https://styleguide.brainly.com/latest/docs/interactive.html?size="small"#round-buttons
+   * @see size="normal" https://styleguide.brainly.com/latest/docs/interactive.html?size="normal"#buttons
+   * @see size="large" https://styleguide.brainly.com/latest/docs/interactive.html?size="large"#buttons
+   */
+  size: $Values<typeof ROUND_BUTTON_SIZE>,
+  /**
+   * Icon types for round buttons
+   * @example <RoundButton iconType="answer" />
+   * @see type="points" https://styleguide.brainly.com/latest/docs/interactive.html?type="points"#round-buttons
+   * @see type="heart" https://styleguide.brainly.com/latest/docs/interactive.html?type="heart"#buttons
+   */
+  iconType?: $Values<typeof TYPE>,
+  /**
+   * Color variants for round buttons
+   * @example <RoundButton color="black-base-500" />
```

Comment:
I introduce the same naming as we do in `style-guide-next` for colors. The plan is to switch to this naming with all the colors in the future. WDYT?

Reply:
but it means in app we will need to use different namings for colors? i am not sure if it's good idea and if it should not be introduced separately.